.github/workflows/check.yaml

```diff
@@ -144,24 +146,30 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        runs-on: [[ubuntu-24.04], [Ubuntu_ARM64_4C_16G_01]]
+        runs-on: [[ubuntu-24.04], [arm64, noble, self-hosted], [s390x, noble, self-hosted], [ppc64el, noble, self-hosted]]
         test-command: ['tox -e func']
-        juju-channel: ["3.4/stable"]
+        juju-channel: ["3.6/stable"]
```

Comment:
Are we ok with upgrading to juju 3.6 for func tests in general? @Pjack

Reply:
I cannot answer it now. But we will track it in https://github.com/canonical/solutions-engineering-automation/issues/227